### ISSUE TITLE
Client crashes when clicking on NEI icons

### ISSUE THREAD

**Issue by notchriss** [ReikaKalseki/Reika_Mods_Issues#790]
I'm assuming this has something to do with RotaryCraft since I can see it mentioned in the exception stack trace 

at Reika.RotaryCraft.ModInterface.NEI.DryingBedHandler.loadCraftingRecipes(DryingBedHandler.java:122)

Client crashes no matter which item you click on, but there's always RotaryCraft, etc. in the stack trace

http://paste.atlauncher.com/view/ee4b8ee5

**Comment by ReikaKalseki** [comment]
What version is this, and why does your mod state block not include my mods?

**Comment by notchriss** [comment]
It's AT launcher, I have no idea how exactly does it log things. This is another log from crash-logs, sorry I didn't think of it earlier, it's been a long week :) http://pastebin.com/FXMt7N58

Your mods versions are:
DragonAPI 1.7.10 V12e
RotaryCraft 1.7.10 V12e

**Comment by ReikaKalseki** [comment]
Something is invalid about your installation. Try removing mods.

**Comment by notchriss** [comment]
Without your mods (Rotary, Reactor, Chromatic, Critter, Electricraft) it works fine. However with only forge and your mods it works fine too. I'll try adding other mods back one by one to see where's the conflict, but it's gonna take a while

Edit: so I removed all mods and re-added them back one by one, deleting old config files for all, and updating as necessary (actually just rftools and mcjtylib needed update). It took about three hours and everything seems to be fine now. Railcraft displayed lots of console errors before I deleted it's old cfg, but it was fine after generating fresh cfg. So I'm guessing it may have had something to do with Railcraft... but that's just a guess. In any case the problem certainly didn't originate in one of your mods.
